Tests/SkillFunctionalTests/FileUpload/FileUploadTests.cs

```diff
@@ -93,14 +93,14 @@ public async Task RunTestCases(TestCaseDataObject testData)
 
             await runner.RunTestAsync(Path.Combine(_testScriptsFolder, testCase.Script), testParams);
 
-            // Create or Update testFile.
-            await using var stream = File.OpenWrite(Directory.GetCurrentDirectory() + $"/FileUpload/Media/{fileName}");
+            // Create a new file to upload.
+            await using var stream = File.Create(Directory.GetCurrentDirectory() + $"/FileUpload/{fileName}");
             await using var writer = new StreamWriter(stream);
             await writer.WriteLineAsync($"GUID:{testGuid}");
             writer.Close();
 
             // Upload file.
-            await using var file = File.OpenRead(Directory.GetCurrentDirectory() + $"/FileUpload/Media/{fileName}");
+            await using var file = File.OpenRead(Directory.GetCurrentDirectory() + $"/FileUpload/{fileName}");
```

Comment:
Any specific reason for moving the file out of 'Media' folder? 🤔

Reply:
I want to delete the Media folder because there's no need to have the file saved there anymore.